src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -25,22 +25,23 @@
  * @author Charles Sarrazin <charles@sarraz.in>
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Antoine Bluchet <soyuka@gmail.com>
+ * @author Baptiste Meyer <baptiste.meyer@gmail.com>
```

Comment:
Why @teohhanhui for me it's okay to add the author name of the people that contributed to the file.